multi_interface_roam/src/multi_interface_roam/multi_interface_roam.py

```diff
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python3
```

Comment:
I believe it's catkin's job to hardcode the version when files are installed with `catkin_install_python`.
Also you did not change the shebang line for any of the other scripts you edited, so please revert it here as well.

Reply:
sure